config.toml

```diff
@@ -13,16 +13,16 @@ thresholds = { 0 = "night", 20 = "dark", 80 = "dim", 250 = "normal", 500 = "brig
 
 [[output.backlight]]
 name = "eDP-1"
-path = "/sys/class/backlight/intel_backlight"
+path = "/sys/devices/pci0000:00/0000:00:08.1/0000:07:00.0/drm/card2/card2-eDP-1/amdgpu_bl2"
 capturer = "wlroots"
 
 # [[output.ddcutil]]
 # name = "Dell Inc. DELL P2415Q"
 # capturer = "none"
 
 [[keyboard]]
-name = "keyboard-dell"
-path = "/sys/bus/platform/devices/dell-laptop/leds/dell::kbd_backlight"
+name = "keyboard-MSI"
+path = "/sys/class/net/enp4s0/enp4s0-2::lan"
```

Comment:
It's okay to keep it for now, but I suppose you want to undo this before final merge :wink: